C++/API/tests.cpp

```diff
@@ -31,6 +31,18 @@ using namespace picojson;
  *
  */
 
+// Constants for running C++ Tests
+const std::string test_project_ID = "1406";
+const std::string test_project_name = "C++ Testing ";
+const std::string test_dataset_name = "Testing ";
+const std::string test_dataset_ID_email = "11366";
+const std::string test_dataset_ID_key = "11367";
+const std::string test_email = "j@j.j";

```

Comment:
Even though these are junk email/passwords they should not be committed to github.